flow/matrix_decomposition.cc

```diff
@@ -1,7 +1,6 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-// FLUTTER_NOLINT
+// found in the LICENSE file.s
```

Comment:
There is an accidental `s` in this line.

Reply:
Removed